ipatests/prci_definitions/nightly_ipa-4-8_latest.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
         git_refspec: '{git_refspec}'
         template: &ci-ipa-4-8-latest
           name: freeipa/ci-ipa-4-8-f32
-          version: 0.0.3
+          version: 0.0.4
```

Comment:
hi @netoarmando I could be wrong but some of the tests in the nightlies might fail with this change.